src/tm/class-wpml-tm-page-builders-field-wrapper.php

```diff
@@ -93,14 +93,25 @@ public function get_string_type() {
 		return $result;
 	}
 
+	/**
+	 * Get string wrap.
+	 *
+	 * @param stdClass $string WPML string.
+	 *
+	 * @return string
+	 */
+	public static function get_wrap( $string ) {
+		return isset( $string->wrap ) ? $string->wrap : '';
+	}
+
 	/**
 	 * @param int $package_id
-	 * @param int $string_id
+	 * @param stdClass $string
 	 *
 	 * @return string
 	 */
-	public static function generate_field_slug( $package_id, $string_id ) {
-		return self::SLUG_BASE . $package_id . '-' . $string_id;
+	public static function generate_field_slug( $package_id, $string ) {
```

Comment:
Do you still need to change the signature? In the end, you stilll use the string ID.

Reply:
@strategio You are right, this is a leftover. Done.